src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -408,6 +409,10 @@ Copyright (c) .NET Foundation. All rights reserved.
                          ILCompilerRuntimeIdentifiers="@(ILCompilerSupportedRids, '%3B')"
                          />
 
+    <KnownILLinkPack Include="Microsoft.NET.ILLink.Tasks"
+                     TargetFramework="net8.0"
+                     ILLinkPackVersion="7.0.100-1.22606.1" />
```

Comment:
Is this version intended to be hard-coded until the linker moves into the runtime repo?